src/openms/include/OpenMS/METADATA/ExperimentalDesign.h

```diff
@@ -62,10 +62,13 @@ namespace OpenMS
     These columns capture the mapping of quantitative values to files for label-free and multiplexed experiments
     and enables fraction-aware data processing.
 
-    - Fraction_Group: a numeric identifier that indicates which fractions are grouped together.
+    - Fraction_Group: a numeric identifier that indicates which fractions are grouped together. Please do NOT
+        reuse the same identifiers across samples! Assign identifiers continuously.
```

Comment:
Just from documentation: so now we allow in addition: 5,6,7,8 ? gaps are not allowed e.g., 5,6,9?

Reply:
I think gaps might actually work. By continuously I meant that you need to continue increasing numbers across samples